src/test/java/spoon/test/UnresolvedBugTest.java

```diff
@@ -45,19 +32,19 @@
 	private final String githubURL = "https://api.github.com/repos/INRIA/spoon/issues/";
 
 
-	@BeforeClass
+	@BeforeAll
 	public static void setup() {
 		testMethods = testMethods.stream()
-				.filter(v -> v.hasAnnotation(Test.class) &&  v.hasAnnotation(Ignore.class)
-						&& v.getAnnotation(Ignore.class).value().equalsIgnoreCase("UnresolvedBug"))
+				.filter(v -> v.hasAnnotation(Test.class) &&  v.hasAnnotation(GitHubIssue.class))
 				.collect(Collectors.toList());
 	}
 
 	/**
 	 * Checks if every githubIssue annotation has an open github issue.
 	 */
-	@Ignore
+	
 	@Test
+	@Disabled
```

Comment:
I'm somewhat skeptical to keeping these tests. I would bet they are absolutely never executed. IMO if it doesn't run in CI, it may as well not exist.